src/routes/twilio/+server.ts

```diff
@@ -94,8 +103,8 @@ export async function POST({ request }: { request: Request }) {
 	}
 
 	// It's a security issue to share the auth link with the client. Don't do this if
-	// import.meta.env.PROD OR private_env.PUBLIC_URL are set
-	if (import.meta.env.DEV && !private_env.PUBLIC_URL) {
```

Comment:
I'm concerned about this check vanishing - If for some reason the env defaults to DEV, I don't want such a major vulnerability to exist. Let's double-check this by ensuring PUBLIC_URL isn't set, or something else that I can configure in production.